.travis.yml

```diff
@@ -4,8 +4,6 @@ node_js:
   - '6'
   - '5'
   - '4'
-  - '0.12'
```

Comment:
Every time I change one of the node tests, I forget that they aren't transpiled, so the CI fails on node < 4. So a bit :+1: on this :smile: